core/common/src/files/FileSystem.kt

```diff
@@ -192,6 +194,7 @@ public expect val SystemTemporaryDirectory: Path
  * Represents information about a file or directory obtainable from a filesystem like
  * a type of filesystem node (is it a directory or a file?), its size, and so on.
  */
+@OptIn(ExperimentalTime::class)
```

Comment:
Corresponding properties has to be annotated with `@ExperimentalTime` instead, but either way having it in the public API is undesirable. So it makes more sense to wait until the Kotlin version is updated to 2.3.